site/docs/design_tokens/colour.mdx

```diff
@@ -138,6 +138,20 @@ Status colours are reserved exclusively for indicating valid, invalid or informa
 | **Black 20**  | black 90      | **AAA** `13,8 : 1`    | <ColorContrast color="var(--color-black-90)" background="var(--color-black-20)">AAA</ColorContrast> | |
 | **Black 10**  | black 90      | **AAA** `15,6 : 1`    | <ColorContrast color="var(--color-black-90)" background="var(--color-black-10)">AAA</ColorContrast> | |
 
+### Focus colour
+HDS components use a custom focus colour styling instead of the default browser provided style. The default HDS focus style is accessible on light backgrounds. Since HDS does not offer dark mode, when using HDS components on darker backgrounds, it may be required to customize the focus style colour. The WCAG requirement for user interface element contrast is `3 : 1`.
+
+| Focus Colour           | Background Colour   | Focus style contrast | Example                                                                                                  |
+| ---------------------- | ------------------- | -------------------- | -------------------------------------------------------------------------------------------------------- |
+| **Vaakuna** (default)  | white               | **AA** `5 : 1`       | <ColorContrast color="var(--color-coat-of-arms-blue)" background="var(--color-white)">AA</ColorContrast> |
+| **Vaakuna** (default)  | black               | **AA** `4,2 : 1`     | <ColorContrast color="var(--color-coat-of-arms-blue)" background="var(--color-black)">AA</ColorContrast> |
+
+It is allowed to customize HDS focus style colour to better fit the theme of the service. However, the implementor is always responsible for ensuring that the following WCAG requirements are met:
```

Comment:
Vähän kiekurainen sanajärjestys. Muotoilisin esim. näin:
"HDS focus style colour can be customised to better fit the theme of the service. However, the implementor must always ensure that the following WCAG requirements are met: